lib/node_modules/@stdlib/fs/append-file/docs/types/test.ts

```diff
@@ -0,0 +1,219 @@
+*/
+
+import appendFile = require( './index' );
+
+const onAppend = ( err: Error | null ): void => {
+	if ( err ) {
+		throw err;
+	}
+};
+
+
+// TESTS //
+
+// The function does not have a return value...
+{
+	appendFile( './beep/boop.txt', 'beepboop', onAppend ); // $ExpectType void
+}
+
+// The compiler throws an error if the function is provided a first argument which is not a string, buffer, or file descriptor...
+{
+	appendFile( false, 'beepboop', onAppend ); // $ExpectError
+	appendFile( true, 'beepboop', onAppend ); // $ExpectError
+	appendFile( null, 'beepboop', onAppend ); // $ExpectError
+	appendFile( undefined, 'beepboop', onAppend ); // $ExpectError
+	appendFile( [], 'beepboop', onAppend ); // $ExpectError
+	appendFile( {}, 'beepboop', onAppend ); // $ExpectError
+	appendFile( ( x: number ): number => x, 'beepboop', onAppend ); // $ExpectError
+}
+
+// The compiler throws an error if the function is provided a second argument which is not a string or buffer...
+{
+	appendFile( './beep/boop.txt', false, onAppend ); // $ExpectError
+	appendFile( './beep/boop.txt', true, onAppend ); // $ExpectError
+	appendFile( './beep/boop.txt', null, onAppend ); // $ExpectError
+	appendFile( './beep/boop.txt', undefined, onAppend ); // $ExpectError
+	appendFile( './beep/boop.txt', [], onAppend ); // $ExpectError
+	appendFile( './beep/boop.txt', {}, onAppend ); // $ExpectError
+	appendFile( './beep/boop.txt', ( x: number ): number => x, onAppend ); // $ExpectError
+}
+
+// The compiler throws an error if the function is provided a callback argument which is not a function with the expected signature...
+{
+	appendFile( '/path/to/beepboop', 'beepboop', 'abc' ); // $ExpectError
+	appendFile( '/path/to/beepboop', 'beepboop', 1 ); // $ExpectError
+	appendFile( '/path/to/beepboop', 'beepboop', false ); // $ExpectError
+	appendFile( '/path/to/beepboop', 'beepboop', true ); // $ExpectError
+	appendFile( '/path/to/beepboop', 'beepboop', null ); // $ExpectError
+	appendFile( '/path/to/beepboop', 'beepboop', undefined ); // $ExpectError
+	appendFile( '/path/to/beepboop', 'beepboop', [] ); // $ExpectError
+	appendFile( '/path/to/beepboop', 'beepboop', {} ); // $ExpectError
+	appendFile( '/path/to/beepboop', 'beepboop', ( x: number ): number => x ); // $ExpectError
+}
+
+// The compiler throws an error if the function is provided an options argument which is not an object or string...
+{
+	appendFile( '/path/to/beepboop', 'beepboop', false, onAppend ); // $ExpectError
+	appendFile( '/path/to/beepboop', 'beepboop', true, onAppend ); // $ExpectError
+	appendFile( '/path/to/beepboop', 'beepboop', null, onAppend ); // $ExpectError
+	appendFile( '/path/to/beepboop', 'beepboop', undefined, onAppend ); // $ExpectError
+	appendFile( '/path/to/beepboop', 'beepboop', 123, onAppend ); // $ExpectError
+	appendFile( '/path/to/beepboop', 'beepboop', [], onAppend ); // $ExpectError
+	appendFile( 
+		'/path/to/beepboop',
+		'beepboop',
+		( x: number ): number => x,
+		onAppend
```

Comment:
Don't do this. Nowhere in the codebase do we split arguments over separate lines.

Reply:
fixed